gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dnumslider_ttt2.lua

```diff
@@ -312,12 +312,15 @@ function PANEL:SetServerConVar(cvar)
 
 	self.serverConVar = cvar
 
-	cvars.ServerConVarGetValue(cvar, function (wasSuccess, value, default)
-		if wasSuccess then
-			self:SetValue(tonumber(value), true)
-			self:SetDefaultValue(tonumber(default))
-		end
-	end)
+	-- Check if self is valid before calling SetValue.
+	if IsValid(self) then
+		cvars.ServerConVarGetValue(cvar, function(wasSuccess, value, default)
+			if wasSuccess then
+				self:SetValue(tonumber(value), true)
+				self:SetDefaultValue(tonumber(default))
+			end
+		end)
+	end
```

Comment:
this here needs to be changed correctly as well

Reply:
and maybe remove the comments, should be self-explanatory

Reply:
You want to be like those two up there, don't you.

Reply:
Yes. Because they have the same structure, they should be the same ^^

Reply:
Does it look like this when it's done?